Dockerfile

```diff
@@ -7,11 +7,11 @@ RUN apt-get update
 RUN apt-get install -y zip
 RUN go get -u -v github.com/ChimeraCoder/gojson/gojson
 RUN go get -u -v github.com/golang/protobuf/protoc-gen-go
```

Comment:
You might be able to remove this now! I deleted it from `.travis.yml` in #527.

Reply:
ah ty, we can indeed kill both of them